### PR TITLE
Nrel building stock

### PR DESCRIPTION
**Pull request by MRossol**
*Issue #, if available:*

*Description of changes:*
Add nrel-pds-building-stock bucket yaml
Add Mid_Atlantic dataset to nrel-pds-wtk
add dataset browsers to all datasets

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/nrel-pds-building-stock.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+UpdateFrequency: Annually
+Collabs:
+  ASDI:
+    Tags:
+      - energy
+Tags:
+  - aws-pds
+  - energy
+  - sustainability
+License:  [ComStock License](https://github.com/NREL/ComStock/blob/main/LICENSE.txt)
```
Comment (jflasher) [inline datasets/nrel-pds-building-stock.yaml:20]: @MRossol I think this should pass if you get rid of the extra space and wrap in quotes.

**Review by jflasher — CHANGES_REQUESTED** [review]

File: datasets/nrel-pds-building-stock.yaml (hunk 1 above)
Comment (MRossol) [reply datasets/nrel-pds-building-stock.yaml]: Thanks @jflasher! I couldn't figure out how to see the CI report, though I was on vacation and not trying very hard... Fix pushed.

**Comment by jflasher** [comment]
@MRossol we had an issue on our side that was causing the explore link regex to fail. Could you pull in the latest and then hopefully, this will pass? Thanks!

File: datasets/wpto-pds-us-wave.yaml [hunk 2]
```diff
@@ -18,38 +18,54 @@ Tags:
   - sustainability
 License: https://github.com/openEDI/documentation/blob/master/US_Wave.md
 Resources:
-  - Description: '[HSDS](https://github.com/NREL/hsds-examples) US Wave domains'
-    ARN: arn:aws:s3:::nrel-pds-hsds/nrel/US_wave/
-    Region: us-west-2
-    Type: S3 Bucket
-  - Description: '[HSDS](https://github.com/NREL/hsds-examples) US Virtual Buoy domains'
-    ARN: arn:aws:s3:::nrel-pds-hsds/nrel/US_wave/virtual_buoy/
+  - Description: 'DOE's Water Power Technology Office's Wave Hindcast datasets'
```
Comment (jflasher) [inline datasets/wpto-pds-us-wave.yaml:21]: "" instead of '' and then 🤞

**Review by jflasher — CHANGES_REQUESTED** [review]

**Comment by jflasher** [comment]
🎉🎉🎉

**Pull request merged**